lib/inferno/apps/cli/execute/quiet_outputter.rb

```diff
@@ -0,0 +1,24 @@
+module Inferno
+  module CLI
+    class Execute
+      # @private
+      class QuietOutputter
+        # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
Don't disable this rule. Prefix the arguments with underscore if they're not used.